Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py

```diff
@@ -304,6 +305,9 @@ def main():
     url = "https://www.threathq.com"
     credentials = params.get("credentials", {})
     auth = (credentials.get("identifier"), credentials.get("password"))
+    if not auth[0] or not auth[1]:
```

Comment:
this is not what i meant
credentials are not mandatory
the code should be something like that:
```python
credentials = params.get("credentials", {}) or {} 
```